### ISSUE TITLE
copy to system clipboard not working

### ISSUE THREAD

**Issue by lcowell** [qvacua/vimr#329]
When I attempt to copy selected text to the system clipboard either by command-C or Edit->Copy, the text is not copied to the clipboard.

I didn't see any open (or closed) issues relating to this so I'm opening this issue. Not sure if this is a just-me or thing or if it is affecting others.

OS X: 10.12.2 Beta (16C32f)
VimR 0.10.0 (118)

**Comment by qvacua** [comment]
That's strange... There was a comment https://github.com/qvacua/vimr/issues/276#issuecomment-253526768 about something similar. Does that help you?

**Comment by lcowell** [comment]
I found the solution. I had added a visual-mode yank remap that seems to have been interfering with how vimr interacts with the yank buffers.

These were the offending lines:

```
vnoremap y myy`y
vnoremap Y myY`y
```

This doesn't seem like it should break VimR's copy functionality. What do you think?

**Comment by qvacua** [comment]
You're right; it should not. The current solution for Cmd-A/C/V expects the default key mappings. However, to stabilize the Edit menu items from possible remappings would require some work, I guess...

**Comment by lcowell** [comment]
I'd say it's a pretty minor issue, but it would be nice to get it fixed one day. Thanks for your help.
